README.md

```diff
@@ -1,20 +1,24 @@
-# A Marriage of Pointer- and Epoch-Based Reclamation
+# General-Purpose SMR Benchmark Suite Written in Rust
```

Comment:
`smr-benchmark`: A microbenchmark suite for concurrent memory reclamation schemes in  Rust